packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -79,6 +93,38 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
       .build();
   }
 
+  public buildQueryForOrgNamespacePrefixCommand(
+    data: IsvDebugBootstrapConfig
+  ): Command {
+    return new SfdxCommandBuilder()
+      .withDescription(
+        nls.localize(
+          'isv_debug_bootstrap_step2_configure_project_retrieve_namespace'
+        )
+      )
+      .withArg('force:data:soql:query')
+      .withFlag('--query', 'SELECT NamespacePrefix FROM Organization LIMIT 1')
```

Comment:
@guw Is the limit clause necessary?

Reply:
In theory it isn't. However, I consider it good practise from my SQL times. You'd like to see it removed?

Reply:
Nope, I was just curious if it was returning more than one.